cost/scheduled_reports/scheduled_report.pt

```diff
@@ -14,10 +14,16 @@ _Note 1: The last 3 days of data in the current week or month will contain incom
 _Note 2: The account you apply the policy to is unimportant as Optima metrics are scoped to the Org._
 
 See [README](https://github.com/flexera/policy_templates/tree/master/cost/scheduled_reports) for more details"
-long_description "Version: 1.8"
+long_description ""
 severity "low"
 category "Cost"
 tenancy "single"
```

Comment:
please add default_frequency "monthly" and update the CHANGELOG that it's been added.

Reply:
Added, Thanks